packages/contracts-bedrock/scripts/upgrades/SafeBuilder.sol

```diff
@@ -57,6 +57,37 @@ abstract contract SafeBuilder is Script {
         return success;
     }
 
+    /**
```

Comment:
Does it make sense to reuse this function as part of the signing codepath?

Reply:
For this, are you suggesting we could use this `computeSafeTransactionHash` as a refactored function inside `_run`?